bindings/python/pyproject.toml

```diff
@@ -4,6 +4,7 @@ build-backend = "maturin"
 
 [project]
 name = "iota-sdk"
+dependencies = ["dacite >= 1.8.1", "pyhumps >= 3.8.0"]
```

Comment:
What's the difference with those in `requirements-dev.txt`?

Reply:
This should also be included in the final package, the requirements-dev.txt not
But have to test first if it actually works this way

Reply:
Works

Reply:
And these are the only required deps ?

Reply:
Yes